ERP-BACKEND/app/routers/health.py

```
@@ -0,0 +1,378 @@
+                "total_connections": pool_stats[0],
+                "active_connections": pool_stats[1],
+                "idle_connections": pool_stats[2]
+            },
+            "timestamp": datetime.now(timezone.utc).isoformat()
+        }
+    except Exception as e:
+        logger.error(f"Database connectivity check failed: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+            detail=f"Database connection failed: {str(e)}"
+        )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)